spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference">
+          <h1>CalendarDateDifference ( _calendar_, _one_, _two_, _largestUnit_ )</h1>
+          <p>The abstract operation CalendarDateDifference performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera">
+          <h1>CalendarDateEra ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEra performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear">
+          <h1>CalendarDateEraYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEraYear performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear">
+          <h1>CalendarDateYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateYear performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth">
+          <h1>CalendarDateMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonth performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthcode">
+          <h1>CalendarDateMonthCode ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthCode performs implementation-defined processing to find the monthCode corresponding to the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateday">
+          <h1>CalendarDateDay ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDay performs implementation-defined processing to find the day for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofweek">
+          <h1>CalendarDateDayOfWeek ( _calendar_, _date_ )</h1>
```

Comment:
Is this an integer between 1-7?  Do we support non-7-day weeks in future built-in calendars? Should clarify expectations.